src/type-translator.ts

```diff
@@ -111,13 +133,43 @@ export function symbolToDebugString(sym: ts.Symbol): string {
     ts.SymbolFlags.ExportType,
     ts.SymbolFlags.ExportNamespace,
     ts.SymbolFlags.Alias,
-    ts.SymbolFlags.Instantiated,
-    ts.SymbolFlags.Merged,
-    ts.SymbolFlags.Transient,
     ts.SymbolFlags.Prototype,
-    ts.SymbolFlags.SyntheticProperty,
-    ts.SymbolFlags.Optional,
     ts.SymbolFlags.ExportStar,
+    ts.SymbolFlags.Optional,
+    ts.SymbolFlags.Transient,
+    ts.SymbolFlags.Enum,
+    ts.SymbolFlags.Variable,
+    ts.SymbolFlags.Value,
+    ts.SymbolFlags.Type,
+    ts.SymbolFlags.Namespace,
+    ts.SymbolFlags.Module,
+    ts.SymbolFlags.Accessor,
+    ts.SymbolFlags.FunctionScopedVariableExcludes,
+    ts.SymbolFlags.BlockScopedVariableExcludes,
+    ts.SymbolFlags.ParameterExcludes,
+    ts.SymbolFlags.PropertyExcludes,
+    ts.SymbolFlags.EnumMemberExcludes,
+    ts.SymbolFlags.FunctionExcludes,
+    ts.SymbolFlags.ClassExcludes,
+    ts.SymbolFlags.InterfaceExcludes,
+    ts.SymbolFlags.RegularEnumExcludes,
+    ts.SymbolFlags.ConstEnumExcludes,
+    ts.SymbolFlags.ValueModuleExcludes,
+    ts.SymbolFlags.NamespaceModuleExcludes,
+    ts.SymbolFlags.MethodExcludes,
+    ts.SymbolFlags.GetAccessorExcludes,
```

Comment:
This list is a bit subtle and maybe not worth keeping right, but it's trying to just be the unique values of the enum.  They have a bunch of stuff in the enum that is just the union of other values in the enum and it seems you've copied them here.

Basically everything up to this line but nothing after
https://github.com/Microsoft/TypeScript/blob/00abefbb683a9317c7f61b11ca7760a938ac199c/src/compiler/types.ts#L2725